shared/components/themes-list/index.jsx

```diff
@@ -44,6 +48,25 @@ var ThemesList = React.createClass( {
 		return 'theme-' + theme.id;
 	},
 
+	getNumColumns: function() {
+		const ThemeListWidth = window.innerWidth - this.getSidebarWidth();
```

Comment:
Could we just get the width of the `theme-list` rect? Using `window` isn't SSR compatible, and will probably cause a forced synchronous layout in browsers, hurting perf: https://developers.google.com/web/fundamentals/performance/rendering/avoid-large-complex-layouts-and-layout-thrashing?hl=en